i18n/README.md

```diff
@@ -0,0 +1,87 @@
+# Internationalization
+
+## Overview
+
+This directory contains the source data files for all internationalization on
+code.org and hourofcode.com. These files should not be directly modified; they
+will be regularly updated by the [i18n sync scripts](../bin/i18n/), which will
+also distribute changes out to the various locations in the repository where
+they will be handled.
+
+## Modifying/Adding a new string
+
+When adding a string, please make the key as descriptive as possible because
+this is used as context for translators.
+
+### Pegasus
+
+#### Modifying an existing string
+
+-   Update the string in the [i18n Gsheet][0]
+-   `ssh staging.code.org` and check that your changes were synced to
+    `staging/pegasus/cache/i18n/en-US.yml`
+-   Commit and push `en-US.yml`
+
+#### Adding a new string
+
+-   Add a unique key and your string value to the [i18n Gsheet][0]. 
+
+    -   NOTE: Make sure your string value only has plain HTML. Organization is
+        by category/page; try to prepend each string of a common category with
+        the same key. For example, all teacher dashboard strings begin with 'dashboard'
+        so they're easier to find.
+
+-   ssh into staging and check that your changes were synced to
+    `staging/pegasus/cache/i18n/en-US.yml`
+
+-   Run the following script to sync your change to all locales.
+
+        bin/i18n/sync-pegasus
+
+-   Add, commit, and push all the files in `i18n/locales` and `pegasus/cache/i18n`.
+
+        git add i18n/locales pegasus/cache/i18n
+
+        git commit -m "sync pegasus strings mm/dd -yourname"
+
+        git push
+
+### Apps
+
+-   Make changes in `code-dot-org/apps/i18n/<app>/en_us.json`
+-   After you push to staging, rake build:apps will run sync-apps.sh and update
+    all locale files.
+-   If you are creating a new app:
+    -   Add the strings in a new file in `code-dot-org/apps/i18n/<new
+        app>/en_us.json`
+    -   Add the app name to the list of "untranslated apps" in
+        `code-dot-org/i18n/code.org/copy-untranslated-apps.rb`
+
+### Dashboard
+
+-   Make changes in `dashboard/config/locales`. Choose from the following files
```

Comment:
We could add `authored_hints`, `callouts`, `courses`, `failure_message_overrides`, and `markdown_instructions`.

Reply:
I actually went ahead and pulled these (and dsls and instructions) out into their own category, to highlight that they are automatically generated from content rather than manually editable

Reply:
Very naaaice